Dockerfile

```diff
@@ -56,7 +56,7 @@ COPY dockerbuild/ssp-overrides/id.php $SSP_PATH/www/id.php
 COPY dockerbuild/ssp-overrides/announcement.php $SSP_PATH/announcement/announcement.php
 COPY tests /data/tests
 
-RUN cp $SSP_PATH/modules/sildisco/sspoverrides/www_saml2_idp/SSOService.php $SSP_PATH/www/saml2/idp/
+RUN cp $SSP_PATH/modules/sildisco/lib/SSOService.php $SSP_PATH/www/saml2/idp/
```

Comment:
This is more for my own education, but why the change from sspoverrides to lib?

Reply:
Unfortunately, I don't remember if it was a requirement or a preference. My understanding as of today is that it's not a good idea to define our own directory name(s) inside a module directory structure. `lib` is defined in SimpleSAMLphp documentation, whereas `sspoverrides` is not. One alternative is to put it in an `sspoverrides` directory (or the existing `dockerbuild/ssp-overrides` at the top level of the repository, but that disconnects it from being associated with the sildisco module. I'm open to suggestions, but I'll go ahead and merge this and implement any changes in a later PR.

Reply:
Okay, that makes sense. As I was reading this PR, it was a little hard to tell where we were copying that file _from_ vs. _to_, and having "sspoverrides" in our _from_ path made that detail more obvious. That's all.